src/main/webapp/creatives/maps/utils.js

```diff
@@ -1,7 +1,7 @@
 
 
 /**
- * Sets the session ID, this should only be used on session creation.
+ * @return {string} A statndard UUID conforming to en.wikipedia.org/wiki/Universally_unique_identifier
```

Comment:
misspelled standard

Reply:
That should have been fixed in an earlier commit. Lmk if you still can't see it